control: improve README how to update submodule (#121)
reazon-research/OpenArm: control/canfd_control/README.md

> git clone --recursive git@github.com:reazon-research/OpenArm.git

Author: I see! Thanks!

> To build, you need to initialize and update `control/canfd_control/vendor/STM32CubeH7` submodule. By cloning the repository with `--recursive` option as shown above, all submodules will be automatically initialized and updated. 

Reviewer: ```suggestion
To build, you need to initialize and update submodules. By cloning the repository with `--recursive` option as shown above, all submodules will be automatically initialized and updated. 
```

Author: I see! Thanks! Fixed.

https://github.com/reazon-research/OpenArm/compare/5368e7dc3318a977bde3fcd35029e819adf948cd..e453efeaf8e6125745b22c470d389131a2d224a8

Alternatively, you can initialize and update submodules after cloning, as shown below.

```bash

> git submodule update --init --recursive control/canfd_control/vendor/STM32CubeH7

Reviewer: ```suggestion
git submodule update --init --recursive
```

Author: I see! Thanks! Fixed.

https://github.com/reazon-research/OpenArm/compare/5368e7dc3318a977bde3fcd35029e819adf948cd..e453efeaf8e6125745b22c470d389131a2d224a8

> Alternatively, you can initialize and update only `control/canfd_control/vendor/STM32CubeH7` submodule after cloning, as shown below.

Reviewer: ```suggestion
Alternatively, you can initialize and update submodules after cloning, as shown below.
```

Author: Thanks!

Author: I thought about `ros2_packages` or `simulation`, but the previous description is confusing, so this seems better.
